api/configuration/v1/kongclusterplugin_types.go

```diff
@@ -40,6 +40,7 @@ import (
 
 // KongClusterPlugin is the Schema for the kongclusterplugins API.
 // +apireference:kic:include
+// +kong:channels=ingress-controller
```

Comment:
Are we going to support it in KGO?

Reply:
That's correct. There's an issue for this here https://github.com/Kong/gateway-operator/issues/524. We decided to postpone it as it was not obvious how to model it so this will require some design work.